cli/dts/lib.deno.worker.d.ts

```diff
@@ -51,20 +53,24 @@ declare class WorkerGlobalScope extends EventTarget {
   Deno: typeof Deno;
 }
 
+/** @category Web APIs */
 declare class WorkerNavigator {
   constructor();
   readonly gpu: GPU;
   readonly hardwareConcurrency: number;
   readonly userAgent: string;
 }
 
+/** @category Web APIs */
 declare var navigator: WorkerNavigator;
 
+/** @category Web Workers */
 interface DedicatedWorkerGlobalScopeEventMap extends WorkerGlobalScopeEventMap {
   "message": MessageEvent;
   "messageerror": MessageEvent;
 }
 
+/** @category Web APIs */
 declare class DedicatedWorkerGlobalScope extends WorkerGlobalScope {
```

Comment:
and here

Reply:
Again, it just contains general APIs, they just happen to be the APIs that are part of the worker scope, not specific to the functionality of web workers.